policy/pa.go

```diff
@@ -130,7 +130,7 @@ var (
 	errInvalidIdentifier   = core.MalformedRequestError("Invalid identifier type")
 	errNonPublic           = core.MalformedRequestError("Name does not end in a public suffix")
 	errICANNTLD            = core.MalformedRequestError("Name is an ICANN TLD")
-	errBlacklisted         = core.MalformedRequestError("Policy forbids issuing for name")
+	errBlacklisted         = core.RejectedIdentifierError("Policy forbids issuing for name")
```

Comment:
Ah, got it! I had missed that part, thanks. What would be the actionable difference for you between the IDN error and the blacklist error? Is it a matter of "revisit later" vs "revisit never?" Trying to figure out what the best error schema is here. For some of these like `errInvalidDNSCharacter` they should be considered a permanent error, but both the IDN error and the blacklist error may be temporary.

What do you think of "rejectedIdentifier" covering blacklisting, and "unsupportedIdentifier" covering IDNs?